src/main/java/com/example/board/domain/member/controller/v2/MemberControllerV2.java

```diff
@@ -0,0 +1,66 @@
+package com.example.board.domain.member.controller.v2;
+
+import com.example.board.common.session.AuthenticatedMember;
+import com.example.board.common.session.SessionManager;
+import com.example.board.domain.member.dto.MemberRequest;
+import com.example.board.domain.member.dto.MemberResponse;
+import com.example.board.domain.member.service.MemberService;
+import java.net.URI;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.util.UriComponentsBuilder;
+
+@RestController
+@RequestMapping("/members/v2")
```

Comment:
```suggestion
@RestController
@RequestMapping("/v2/members")
```
요렇게해야 자원을 잘 표현할 수 있을 것 같아요